src/screens/SearchBeers/SearchBeers.jsx

```diff
@@ -137,172 +138,171 @@ const maltOptions = [
   }
 ]
 
-class SearchBeersComponent extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      beers: [],
-      alcoholVolume: '',
-      ibuRange: '',
-      colorOfBeer: '',
-      brewedBefore: '',
-      food: '',
-      beerName: '',
-      yeast: '',
-      hops: '',
-      malt: '',
-    };
-  };
+function SearchBeersComponent() {
+
+  const [beers, setBeers] = useState([])
+  const [params, setParams] = useState({
+    alcoholVolume: '',
+    id: '',
+    ibuRange: '',
+    colorOfBeer: '',
+    brewedBefore: '',
+    food: '',
+    yeast: '',
+    hops: '',
+    malt: '',
+    beerName: '',
+  });
+
+  function handleChange(event) {
 
-  handleChange = event => {
     const { name, value } = event.target;
-    this.setState({
-      [name]: value
-    });
+    setParams(prevParam => (
+      { prevParam, [name]: value })
+    );
   };
 
-  getBeers = async () => {
-    const { alcoholVolume, ibuRange, colorOfBeer, brewedBefore, food, yeast, hops, malt } = this.state;
-    const beers = await fetchBeers([alcoholVolume, ibuRange, colorOfBeer, brewedBefore, food, yeast, hops, malt]);
-    this.setState({ beers });
+  async function getBeers() {
+    const beers = await fetchBeers([params.alcoholVolume, params.ibuRange, params.colorOfBeer, params.brewedBefore, params.food, params.yeast, params.hops, params.malt]);
+    setBeers(beers);
   };
 
-  getBeerByName = async () => {
-    const { beerName } = this.state
-    const beers = await fetchBeers([`beer_name=${beerName}`]);
-    this.setState({ beers })
-  }
+  async function getBeerByName() {
```

Comment:
Do you fetch single beer or more than 1?

Reply:
Fixed!